Library/.rubocop.yml

```diff
@@ -1,4 +1,5 @@
 AllCops:
+  AllowSymlinksInCacheRootDirectory: true
```

Comment:
Weird, now it is working even without it. I had to set it because RuboCop complained about `/tmp` being a symlink since the cache directory while running tests is inside `/tmp`.